server/config_libpod.go

```diff
@@ -0,0 +1,24 @@
+// +build libpod
+
+package server
+
+import (
+	"fmt"
+	"github.com/containers/storage"
+	"github.com/projectatomic/libpod/libpod/image"
+)
+
+// ImageRuntime returns a libpod image.runtime to access the image store
+func (s *Server) ImageRuntime() *image.Runtime {
+	options := storage.StoreOptions{
+		RunRoot:   s.Config().RunRoot,
+		GraphRoot: s.Config().RootConfig.Root,
+	}
+	r, err := image.NewImageRuntimeFromOptions(options)
+	// For now, if we get an error trying to get the imageruntime, we panic. this can be addressed
+	// as integration continues.
+	if err != nil {
+		panic(fmt.Sprintf("unable to get libpod image runtime: %s", err.Error()))
```

Comment:
can't we change the whole `ImageRuntime` signature to return an error?

Reply:
> can't we change the whole `ImageRuntime` signature to return an error?

The plan is to run the error-producing portions at startup, once the libpod stuff becomes the default.  Then we can `sed` to replace calls to this getter with struct property access.  That `sed` is easier if this getter panics instead of returning an error.

[1]: https://github.com/kubernetes-incubator/cri-o/pull/1669#discussion_r199630923